language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -193,55 +174,65 @@ tests damlTypes yarn damlc daml2ts davl = testGroup "daml2ts tests"
         buildProject []
       withCurrentDirectory here $ do
         step "daml2ts..."
-        setupWorkspace
-        daml2tsProject [groverDar, groverDar] daml2tsDir (here </> "package.json")
+        writePackageJson
+        copyDirectory damlTypes "daml-types"
+        daml2tsProject [groverDar, groverDar] daml2tsDir
         assertFileExists (groverTsSrc </> "Grover.ts")
+        assertFileExists (groverTsLib </> "Grover.js")
+        assertFileExists (groverTsLib </> "Grover.d.ts")
         assertFileExists (groverTsSrc </> "packageId.ts")
+        assertFileExists (groverTsLib </> "packageId.js")
+        assertFileExists (groverTsLib </> "packageId.d.ts")
```

Comment:
We should also check for `index.*`. It might make sense to have a function like
```haskell
assertTsFileExists dir file = do
  assertFileExists (dir </> "src" </> file <.> "ts")
  assertFileExists (dir </> "lib" </> file <.> "js")
  assertFileExists (dir </> "lib" </> file <.> "d.ts")
```
and call
```haskell
mapM_ (assertTsFileExists groverTs) ["index", "Grover", "packageId"]
```
That would be beneficial below as well.

Reply:
This is a very nice idea. Done.